### DISCUSSION TITLE
Spectate button in multiplayer should auto download currently playing beatmap

### DISCUSSION THREAD

**Discussion by 0xflotsky** [ppy/osu#29634] [Ideas] [answered]
In lazer multiplayer when you press Spectate and you don't have the currently playing beatmap you will get to 'Spectating' state but the map won't start downloading and you would just keep waiting for nothing to happen.

Would make sense that if you press Spectate it should start downloading the map if you don't have it

**Comment by peppy** [comment]
Have you tried turning on auto downloads?

**Reply by 0xflotsky** [reply]
it's on, doesn't help in multiplayer lobbies

**Reply by peppy** [reply]
Seems like something we should fix then 😄

**Reply by peppy** [reply, marked as answer]
#29680
